Src/BootCamp.Chapter/Demo.cs

```diff
@@ -15,22 +15,22 @@ public sealed class Demo
         public static event EventHandler<DemoAppEventArgs> AppStatusChanged;
 
         private static readonly ContactsCenter contactsCenter = new ContactsCenter("Input/MOCK_DATA.csv");
-        private static readonly Menu menu = new Menu("Main menu", PopulateMainMenu());
+        private readonly Menu menu = new Menu("Main menu", PopulateMainMenu());
 
         public void Run()
         {
             menu.MenuItemSelectedEvent += OnMenuItemSelected;
-            OnAppStatusChanged(AppStatus.DemoStarted);
+            StatusChanged(AppStatus.DemoStarted);
```

Comment:
This is actually not needed as a method, you just call another line under it. ChangeStatus would be a better name, because now it looks like you invoke an event (which you don't)